src/tools/rbd/CMakeLists.txt

```diff
@@ -58,13 +58,14 @@ set(rbd_srcs
 add_executable(rbd ${rbd_srcs}
   $<TARGET_OBJECTS:common_texttable_obj>)
 set_target_properties(rbd PROPERTIES OUTPUT_NAME rbd)
-target_link_libraries(rbd librbd
+target_link_libraries(rbd
+  librados
```

Comment:
Since it should be left-to-right in terms of most to least dependent, why would `librados` be listed first instead of after `libneorados`?